Java/app/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.1.0'
 
     //Taboola
-    implementation 'com.taboola:android-sdk-beta:meta-unified-1.0.0-570-8bc74b127'
+    implementation 'com.taboola:android-sdk-beta:meta-twenty-minutes-beta-2-598-dea2d0cb0'
```

Comment:
@voltfmanSlava - Any reason we are using a specific version of a publisher here instead of a unified version?

Reply:
@tomer-br-taboola on that moment i added it there wasn't unified with the code. will update to the latest unified.

Reply:
Done